Framework/Blockchain/Ethereum/Account.cs

```diff
@@ -1,59 +1,30 @@
 ﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.IO;
-using UnityEngine;
 using Nethereum.Hex.HexTypes;
 using Nethereum.JsonRpc.UnityClient;
 using Nethereum.RPC.Eth.Blocks;
-using System.Threading.Tasks;
 using Nethereum.RPC.TransactionManagers;
 using Nethereum.Signer;
 
 namespace Hyperbridge.Blockchain.Ethereum
 {
     public class Account
     {
-        public string Address { get; protected set; }
-        public string PrivateKey { get; protected set; }
-
-#if !PCL
-        public static Account LoadFromKeyStoreFile(string filePath, string password)
-        {
-            var keyStoreService = new Nethereum.KeyStore.KeyStoreService();
-            var key = keyStoreService.DecryptKeyStoreFromFile(password, filePath);
-
-            return new Account(key);
-        }
-#endif
-
-        public static Account LoadFromKeyStore(string json, string password)
-        {
-            var keyStoreService = new Nethereum.KeyStore.KeyStoreService();
-            var key = keyStoreService.DecryptKeyStoreFromJson(password, json);
-
-            return new Account(key);
-        }
+        public string Address { get; }
+        public string PrivateKey { get; }   // Should the private key really be public?
```

Comment:
Nope, it shouldn't be accessible directly. It should have a method for signing transactions.